btk/draw_blends.py

```diff
@@ -291,59 +340,41 @@ def render_blend(self, blend_catalog, cutout, filt):
 
         Args:
             blend_catalog: Catalog with entries corresponding to one blend.
-            cutout: `btk.obs_conditions.Cutout` class describing observing conditions.
             filt(string): Name of filter to draw images in.
 
         Returns:
             Images of blend and isolated galaxies as `numpy.ndarray`.
 
         """
-        if not hasattr(cutout, "survey"):
-            mean_sky_level = cutout.mean_sky_level
-        elif not hasattr(cutout, "mean_sky_level"):
-            mean_sky_level = cutout.survey.mean_sky_level[
-                [b == filt for b in cutout.survey.filters]
-            ]
-        else:
-            raise AttributeError("cutout needs a `survey`  as an attribute.")
-
+        mean_sky_level = get_mean_sky_level(survey, filt)
         blend_catalog.add_column(
             Column(np.zeros(len(blend_catalog)), name="not_drawn_" + filt.name)
         )
-
-        pix_stamp_size = np.int(self.stamp_size / cutout.pixel_scale)
+        pix_stamp_size = np.int(self.stamp_size / survey.pixel_scale)
```

Comment:
```suggestion
        pix_stamp_size = int(self.stamp_size / survey.pixel_scale)
```

Deprecated in numpy 1.20, should be replaced with pure `int`

Reply:
thanks